datasets/sk/rpvs/crawler.py

```diff
@@ -1,14 +1,211 @@
-import zipfile
+from typing import Any
 
-from zavod import Context
-from zavod.shed.bods import parse_bods_fh
+from zavod import Context, helpers as h
 
+BASE_URL = "https://rpvs.gov.sk/opendatav2"
+ENTITY_DETAILS = (
+    f"{BASE_URL}/PartneriVerejnehoSektora/{{id}}?$expand=Partner,PravnaForma,Adresa"
+)
+PARTNER_DETAILS = f"{BASE_URL}/Partneri/{{id}}?$expand=Vymaz,Pokuta,OverenieIdentifikacieKUV,konecniUzivateliaVyhod,verejniFunkcionari,kvalifikovanePodnety"
+BENEFICIAL_OWNERS = (
+    f"{BASE_URL}/KonecniUzivateliaVyhod/{{id}}?$expand=Partner,PravnaForma,Adresa"
+)
+PUBLIC_OFFICIALS = f"{BASE_URL}/VerejniFunkcionari/{{id}}"
 
-def crawl(context: Context) -> None:
-    fn = context.fetch_resource("source.zip", context.data_url)
-    with zipfile.ZipFile(fn, "r") as zf:
-        for name in zf.namelist():
-            if not name.endswith(".json"):
-                continue
-            with zf.open(name, "r") as fh:
-                parse_bods_fh(context, fh)
+TOTAL_COUNT = "https://rpvs.gov.sk/opendatav2/PartneriVerejnehoSektora/$count"
+FIRST_PAGE = "https://rpvs.gov.sk/opendatav2/PartneriVerejnehoSektora?$skip=0"
+IGNORE_FIELDS = [
+    # @odata.context is a metadata URL
+    "@odata.context",
+    # Start and end dates of validity might not be the same as the dates of incorporation and dissolution
+    "valid_from",
+    "valid_to",
+]
+
+# TODO do we want to add contry "SK" everywhere?
```

Comment:
Do we want to do this? You'll know better, but I guess `country` is less specific than nationality, just saying that there is some association with Slovakia? Your call :)